build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/JdkDownloadPlugin.java

```diff
@@ -108,7 +108,7 @@ private void setupRepository(Project project, Jdk jdk) {
                     + "/[module]/[classifier]/jdk/hotspot/normal/adoptium";
             }
         } else if (jdk.getVendor().equals(VENDOR_OPENJDK)) {
-            repoUrl = "https://download.oracle.com";
```

Comment:
How come we used download.oracle.com? this seems not to work (anymore) I switched this to download.java.net